src/cmd-buildextend-installer

```diff
@@ -92,7 +92,7 @@ os.mkdir(tmpisoisolinux)
 initrd_ignition_padding = 256 * 1024
 
 def generate_iso():
-    arch = platform.machine()
+    arch = get_basearch()
```

Comment:
How about renaming these vars to `basearch`/`BASEARCH` to be more explicit?

Reply:
I'll leave that to a followup: https://github.com/coreos/coreos-assembler/pull/799